README.md

```diff
@@ -411,6 +411,8 @@ The following settings are optional and allow you to further configure your clus
 * `settings.kubernetes.topology-manager-policy`: Specifies the topology manager policy. Possible values are `none`, `restricted`, `best-effort`, and `single-numa-node`. Defaults to `none`.
 * `settings.kubernetes.topology-manager-scope`: Specifies the topology manager scope. Possible values are `container` and `pod`. Defaults to `container`. If you want to group all containers in a pod to a common set of NUMA nodes, you can set this setting to `pod`.
 * `settings.kubernetes.pod-pids-limit`: The maximum number of processes per pod.
+* `settings.kubernetes.image-gc-high-threshold-percent`: The disk utilization percentage where `kubelet` begins garbage collection over the stored container images. In short, this is the "disk is too full" mark.
```

Comment:
This doesn't sound correct; my reading of the documentation is that the garbage collector will run less frequently - not always, but not never - if disk usage percentage falls between high and low.

I would also avoid phrases like "In short". Maybe just strike the second sentence from each since you're introducing an abstract concept - whether or not the disk is too full or empty enough - to what would otherwise be a precise description of how the setting controls the kubelet garbage collector.

Reply:
You are correct. It seems like the garbage collection follows a periodic cleanup (every minute for containers, every five minutes for images). The High and Low threshold limits are used in the following way:

> Disk usage above the configured HighThresholdPercent value triggers garbage collection, which deletes images in order based on the last time they were used, starting with the oldest first. The kubelet deletes images until disk usage reaches the LowThresholdPercent value.

So, if the High threshold is exceeded, garbage collection starts and does not stop until the Low threshold is reached. Then, the normal periodic cleanup behavior resumes.

@bcressey  What do you think of the following for `settings.kubernetes.image-gc-high-threshold-percent`?

```The disk utilization percentage which, when exceeded, `kubelet` switches from periodic garbage collection to constant garbage collection, until the disk utilization percentage hits `settings.kubernetes.image-gc-low-threshold-percent`.```

And for `settings.kubernetes.image-gc-low-threshold-percent`:
```The disk utilization percentage where `kubelet` stops constant garbage collection and returns to periodic garbage collection.```